cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1278,7 +1278,7 @@ def test_move_and_discard_changes(self, store_type):
         self.assertFalse(self.store.has_changes(self.store.get_item(old_parent_loc)))
 
         # Move component html_usage_key in vert2_usage_key
-        self.assert_move_item(self.html_usage_key, self.vert2_usage_key)
+        self.assert_move_item(self.html_usage_key, self.vert2_usage_key, 0)
```

Comment:
Otherwise test fails. 
Without any value variable initialised with Nonetype and raises 400.